bot/util/utilities.py

```diff
@@ -1,29 +1,158 @@
+
+        Args:
+            ctx (Context): The context in which the command was called.
 
-        :param ctx: the context in which the command was called
+        Returns:
+            str: A message containing 'Pong!', as well as the measured latency to the Discord server in milliseconds.
         """
         latency = round(self.bot.latency * 1000, 2)
-        await ctx.send("\U0001F3D3 **Pong!** - {0} ms".format(latency))
+        await ctx.send(":ping_pong: **Pong!** - {0} ms".format(latency))
+
+    @commands.command(name='serverinfo')
+    async def server_info(self, ctx):
+        """Command Handler for the `serverinfo` command.
+
+        Args:
+            ctx (Context): The context in which the command was called.
+
+        Returns:
+            Embed: An embedded message (Embed) containing a variety of stats and interesting information about the
+                    Discord server.
+        """
+        str_owner = "{0.display_name}#{0.discriminator}" \
+            .format(ctx.guild.owner)
+        str_boosts = "__Level {0.premium_tier}__\n{0.premium_subscription_count}{1} Boosts" \
+            .format(ctx.guild, determine_boost_level_cap(ctx.guild.premium_subscription_count))
+        str_members = "__Gesamt: {0[0]}__\nBots: {0[1]}\nMenschen: {0[2]}" \
+            .format(get_member_counters(ctx.guild))
+        str_channels = "__Gesamt: {0[0]}__\nText: {0[1]}\nSprach: {0[2]}" \
+            .format(get_channel_counters(ctx.guild))
+        str_roles = "__Gesamt: {0}__" \
+            .format(len(ctx.guild.roles))
+        str_features = generate_features_string(ctx.guild.features)
+
+        embed = discord.Embed(title=ctx.guild.name, timestamp=datetime.now(), color=0x1E90FF)
+        embed.set_thumbnail(url=ctx.guild.icon_url)
+        embed.set_footer(text="Erstellungsdatum")
+        embed.add_field(name="Besitzer :crown:", value=str_owner, inline=True)
+        embed.add_field(name="Server Boost <:server_boost:730390579699122256>", value=str_boosts, inline=True)
+        embed.add_field(name="Server Features :tools:", value=str_features, inline=True)
+        embed.add_field(name="Mitglieder :man_raising_hand:", value=str_members, inline=True)
+        embed.add_field(name="Kanäle :dividers:", value=str_channels, inline=True)
+        embed.add_field(name="Rollen :medal:", value=str_roles, inline=True)
+        await ctx.send(embed=embed)
+
+
+def determine_boost_level_cap(amount_boosts):
+    """Function for determining the current server level cap.
+
+    Args:
+        amount_boosts (int): The number of Boosts a server has received from its members.
+
+    Returns:
+        str: A short message indicating how many boosts are needed to level up.
+    """
+    level_cap = ""
+
+    if amount_boosts < 2:
+        level_cap = "/2"
+    elif 2 <= amount_boosts < 15:
+        level_cap = "/15"
+    elif 15 <= amount_boosts < 30:
+        level_cap = "/30"
+
+    return level_cap
+
+
+def get_channel_counters(guild):
```

Comment:
You don't need to change this, but I want to mention that this method violates separation of concern, bc this function does 3 things, but should only do one.